routing/router.go

```diff
@@ -847,11 +850,12 @@ func (r *ChannelRouter) FindRoutes(target *btcec.PublicKey, amt btcutil.Amount)
 	return validRoutes, nil
 }
 
-// generateSphinxPacket generates then encodes a sphinx packet which encodes
+// generateOnionBlob generates then encodes a sphinx packet which encodes
```

Comment:
Why the rename? Diffs should be kept as minimal as possible. Please revert. Thanks!